src/FixedPointDecimals.jl

```diff
@@ -348,24 +342,24 @@ Base.zero(::Type{FD{T, f}}) where {T, f} = reinterpret(FD{T, f}, zero(T))
 Base.one(::Type{FD{T, f}}) where {T, f} = reinterpret(FD{T, f}, coefficient(FD{T, f}))
 
 # comparison
-==(x::T, y::T) where {T <: FD} = x.i == y.i
- <(x::T, y::T) where {T <: FD} = x.i  < y.i
-<=(x::T, y::T) where {T <: FD} = x.i <= y.i
+Base.:(==)(x::T, y::T) where {T <: FD} = x.i == y.i
+Base.:( <)(x::T, y::T) where {T <: FD} = x.i  < y.i
```

Comment:
I say we just drop the whitespace padding here
```suggestion
Base.:(<)(x::T, y::T) where {T <: FD} = x.i < y.i
```

Reply:
Done. Thanks